ManageIQ/manageiq-ui-classic
# Support for VMware's new WebMKS HTML consoles in the UI

This is required as the VMware 6.5 and above no longer supports VNC and WebMKS is the preferred way of opening HTML5-based consoles.

Related issue: https://github.com/ManageIQ/manageiq/issues/13798
Depends on: https://github.com/ManageIQ/manageiq/pull/13945

Steps to test:
1. Have a VM from a VMware provider v6.0 or newer
1. Make sure that the SSL certificate errors are ignored or the provider's certificate is on a whitelist
1. Go to *Configure* -> *Configuration* -> *VMware Console Support* and set it to *VMware WebMKS*
1. Go to the VM's summary screen
1. Select *Access* -> *VM Console*

@miq-bot add_label pending core, euwe/no
@miq-bot assign @martinpovolny 

/cc @Chuk-i-Gek for the new toolbar button
